tests/electrum.rs

```diff
@@ -125,7 +125,7 @@ fn test_electrum() -> Result<()> {
         )?]),
     )?;
     notify_wallet();
-    assert_balance(0.3, -0.161);
+    assert_balance(0.139, 0.0);
```

Comment:
This change is required to upgrade electrum from 4.1.5 to 4.5.4

Reply:
Possibly related to https://github.com/spesmilo/electrum/pull/7807